cmd/pipeline/config.go

```diff
@@ -172,8 +190,4 @@ func configure(v *viper.Viper, p *pflag.FlagSet) {
 
 	v.SetDefault("spotmetrics::enabled", false)
 	v.SetDefault("spotmetrics::collectionInterval", 30*time.Second)
-
-	v.SetDefault("audit::enabled", true)
-	v.SetDefault("audit::headers", []string{"secretId"})
```

Comment:
**Note**: this configuration option seems to be completely removed from the new code. This is just an FYI notice to make sure this decision was deliberate and has no problematic consequences - I don't see any, but my domain knowledge is heavily lacking to judge it properly.

Reply:
We don't use that header anymore, as such, whitelisting it doesn't make any sense. But you are right, this removes the header logging functionality.